docs/troubleshooting.md

```diff
@@ -24,6 +37,79 @@ The Open vSwitch daemon logs for each `antrea-agent` Pod are also stored on the
 persistent storage of the corresponding node (i.e. the node on which the Pod is
 scheduled), under `/var/log/antrea/openvswitch`.
 
+## Accessing the antrea-controller API
+
+antrea-controller runs as a Deployment, exposes its API via a Service and
+registers an APIService to aggregate into the Kubernetes API. To access the
+antrea-controller API, you need to know the address of it and have credentials
+to access it. There are multiple ways you can do it.
+
+### Using antctl
+
+Typically, `antctl` handles locating and authenticating when it runs in an
+environment with kubeconfig set up. Same as `kubectl`, `antctl` looks for a
+file named config in the $HOME/.kube directory. You can specify other
+kubeconfig files by setting the `--kubeconfig` flag.
```

Comment:
independent of this PR, but does `antctl` support the `KUBECONFIG` env variable as well?

Reply:
It doesn't support yet, we should do it too to make antctl can run anywhere kubectl env is well configured.